### ISSUE TITLE
voxel_vote: command not found, disk quota, overlap issue

### ISSUE THREAD

**Issue by gk1250** [CobraLab/MAGeTbrain#61]
Hi, I have some questions about the errors in running the magetbrain.

salloc --x11 -p day -t 1- -c 36 --mem-per-cpu 20g
source MAGeTbrain/bin/activate
module load parallel
module load Python/3.10.8-GCCcore-12.2.0
source MINC2/minc-toolkit-config.sh
export QBATCH_PPJ=8
export QBATCH_CHUNKSIZE=1
export QBATCH_CORES=1
export QBATCH_MEM=32G
mb status
mb run --stage-templatelib-procs 1 --stage-voting-procs 8 

These are the commands I used to run the magetbrain.

I was wondering if "/usr/bin/sh: voxel_vote: command not found" is causing the disk quota to exceed. I keep getting 500+ mini files in the output/intermediate directory (outside the template folder, nothing in the fusion folder) and I'm not sure it's normal. If it's an error, could you tell me how to fix it? Would incorporating files you had uploaded in the antsRegistration-MAGeT/bin folder such as mb_vote.sh be helpful in fixing this issue? 

Also, I manually did AC-PC alignment for each subject's T1 MPRAGE image (RAS oriented, skull stripped, no smoothing, no co-registration), and I got this error message: 
MattesMutualInformationImageToImageMetrics4(0x2c97790): Too many samples map outside moving image buffer. There are only 7 valid points out of 21384 total points. The images do not sufficiently overlap. They need to be initialized to have more overlap before this metric will work. For instance, you can align the image centers by translation. 
Would ignoring this error message be okay?

Thank you so much!

**Comment by gdevenyi** [comment]
You have not installed all the dependencies.

https://github.com/cobralab/MAGeTbrain#dependencies

As for alignment, you should be performing _no_ prealignment at all, simply inputting native-space images. Performance is better with preprocessing and skull-stripping but not required.

I suggest you take a look at https://github.com/cobralab/MAGeTbrain-nextflow which implements a nextflow implementation with containerized runtime and supports nifti files for all inputs. I'll try and get around to adding some basic instructions, you can take a look at the nextflow.

Finally, you have not described at all the input configuration you are using, the number of files, the atlases, labels, their origins, etc. PLease do so for any further support requests.

**Comment by gk1250** [comment]
Hi Dr.Devenyi,


Thank you for your email.

I'm using 5 habenula segments that are publicly available on the CoBrALab
GitHub page for the atlases and 50 children's T1 MPRAGE data for the
subjects (I selected 21 of them with the lowest average motion for the
templates).

As you suggested, I ran the program again with skull-stripped data and no
prealignment, but got the same error message.
I got this error message for 18 participants' data, and somehow they are
all part of the templates folder.
[image: overlap error.png]
I was wondering if there are ways to handle this error or if it would be
okay to ignore it and continue with the voting process.

Thank you so much for your help.


Best regards,
Gihyun

ᐧ
